packages/hawtio/src/plugins/camel/routes/CamelRoutes.tsx

```diff
@@ -215,7 +220,9 @@ export const CamelRoutes: React.FunctionComponent = () => {
                   }}
                 />
                 <Td dataLabel='Name'>{route.routeId}</Td>
-                <Td dataLabel='State'>{route.state}</Td>
+                <Td dataLabel='State'>
+                  <RouteStateLabel state={route.state!} />
```

Comment:
Why `state!`?  I believe it's a non-null assertion, but you don't seem to really assert the state is not null nor undefined. Instead, isn't it better to provide a default value with `route.state ?? <something>`?

Reply:
you,re right. Or default unknown state in the Label component.

Reply:
Or simply leave it empty?

Reply:
e.g. `{route.state ? <RouteStateLabel state={route.state} /> : ''}`

Reply:
Well I'd like to have a gray unknown label instead  😇